src/linagora.esn.contact/app/services/contact-dav-client.service.js

```diff
@@ -0,0 +1,52 @@
+
+angular.module('linagora.esn.contact')
+  .factory('contactDavClientService', contactDavClientService);
+
+function contactDavClientService($http, $q, tokenAPI, contactDAVURLService) {
+  let cachedDavURLPromise = null;
```

Comment:
It is easier to read if you consider URL as a word, so in camel case `cachedDavURLPromise` should be `cachedDavUrlPromise`. You are already doing this for DAV.
This is no the only place in this code where you wrote URL with uppercase letters (see https://github.com/linagora/esn-frontend-contacts/pull/172/files#diff-bb24ac631cc25ccf19ba430411465789ba800ebdb7f68ed4e56eb1e676320aa3R4)